aleph/core.py

```diff
@@ -46,7 +46,11 @@
 def determine_locale():
     try:
         options = SETTINGS.UI_LANGUAGES
-        locale = request.accept_languages.best_match(options)
+        if str(request.accept_languages) == "*":
```

Comment:
Just a quick note: I haven’t tested this, but I saw that `best_match` has a default argument which is returned if none of the options matches the `Accept-Language` header. If that covers this case as well, it might be a little bit more robust as it would also cover cases where `Accept-Language` contains a list of languages but none of them matches one of the options.

https://github.com/pallets/werkzeug/blob/main/src/werkzeug/datastructures/accept.py#L148

Reply:
Unfortunately it does not cover this case, because it immediately exits with the first found value. But I can add the default language to the `best_match` invocation for good measure.

Reply:
(done)

Reply:
Nice, thanks for checking!